diff --git a/node_modules/react-native/ReactCommon/react/renderer/core/graphicsConversions.h b/node_modules/react-native/ReactCommon/react/renderer/core/graphicsConversions.h
index bb59351..e67ae3b 100644
--- a/node_modules/react-native/ReactCommon/react/renderer/core/graphicsConversions.h
+++ b/node_modules/react-native/ReactCommon/react/renderer/core/graphicsConversions.h
@@ -9,6 +9,7 @@
 
 #include <array>
 #include <unordered_map>
+#include <sstream>
 
 #include <glog/logging.h>
 #include <react/debug/react_native_expect.h>
@@ -76,8 +77,11 @@ inline folly::dynamic toDynamic(const YGValue& dimension) {
       return "stretch";
     case YGUnitPoint:
       return dimension.value;
-    case YGUnitPercent:
-      return std::format("{}%", dimension.value);
+    case YGUnitPercent: {
+      std::ostringstream stream;
+      stream << dimension.value << '%';
+      return stream.str();
+    }
   }
 
   return nullptr;
